tsconfig.json

```diff
@@ -5,14 +5,10 @@
         "target": "ES6"
     },
     "include": [
-        "*.ts",
         "**/*.ts"
     ],
     "exclude": [
-        "2",
-        "3",
         "js",
-        "ignore",
         "node_modules"
```

Comment:
Those extra lines for TS exclusion are there because I have directories on my website at https;//prettydiff.com that I don't want TypeScript looking into.